.changeset/nice-rules-fetch.md

```diff
@@ -0,0 +1,7 @@
+---
+'@chromatic-com/playwright': patch
+'@chromatic-com/cypress': patch
+'@chromatic-com/shared-e2e': patch
```

Comment:
I assume this is the right way to do this -- since the `shared-e2e` package changed, I assume all 3 packages should have a version bump.

Reply:
Yeah we haven't really talked about that. It doesn't matter since we don't publish the shared version and the ones we do publish depend on whatever the current version is with `workspace:*`. So we only need to bump it if we want to keep its changelog updated. But it's really the external packages where the changelog matters so I don't know what we want to do. Let's sync up about our approach.